config/locales/en.yml

```diff
@@ -424,6 +429,25 @@ en:
       muting: Muting list
     upload: Upload
   in_memoriam_html: In Memoriam.
+  invites:
+    delete: Delete
+    expires_in:
+      '1800': 30 minutes
+      '21600': 6 hours
+      '3600': 1 hour
+      '43200': 12 hours
+      '86400': 1 day
```

Comment:
shouldn't this just be done with an interval thing?

Reply:
No, we overwrote the rails strings to be super short for relative time. Unless you know a better way